src/back-end/lib/resources/affiliation.ts

```diff
@@ -131,40 +124,38 @@ const resource: Resource = {
 
   update(connection) {
     return {
-      async parseRequestBody(request) {
+      async parseRequestBody(request): Promise<null> {
```

Comment:
You shouldn't need to type the return value of this function, as it should be inferred by the `Resource` type above.